jest.config.ts

```diff
@@ -14,6 +14,7 @@ export default {
     __TEST__: true,
   },
   moduleNameMapper: {
+    '^@dinero.js/test$': '<rootDir>/test/utils/',
```

Comment:
I'm afraid this could be confusing for contributors who discover the codebase and don't check this file, because it has exactly the same scheme as legitimate dependencies.

We could keep it simple and just map this path to `'test-utils'`, wdyt?